### PR TITLE
Fix to support for non 'us' keys,  Add support to Global KMS keys and Key update

### PR DESCRIPTION
**Pull request by lcaggio**
- Add support for non 'us' regions
 - Keep disk device name
 - Add timestamp to keep disk name unique
 - Add support for Cloud KMS global keys

### CODE REVIEW

File: tools/gce-google-keys-to-cmek/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-# Google-managed key to CMEK
+# Update GCE Disks Endription Key in use
```
Comment (rosmo) [inline tools/gce-google-keys-to-cmek/README.md:1]: Typo (encryption)?

File: tools/gce-google-keys-to-cmek/main.py [hunk 2]
```diff
@@ -101,28 +109,32 @@ def migrate_instance_to_cmek(project, zone, instance, key_ring, key_name,
   for source_disk in disks:
     disk_regexp = r'^https:\/\/www\.googleapis\.com\/compute\/v1\/projects\/(.*?)\/zones\/(.*?)\/disks\/(.*?)$'
```
Comment (ludoo) [inline tools/gce-google-keys-to-cmek/main.py:110]: I would move the pattern to a module-level constant (and maybe use split instead of a regexp anyway)

**Review by ludoo — APPROVED** [review]

**Pull request merged**